lib/utils.js

```diff
@@ -81,6 +81,27 @@ function validate_metadata(object, name) {
 }
 
 /**
+ * Create base url for registry.
+ * @param {String} protocol
+ * @param {String} host
+ * @param {String} prefix
+ * @return {String} base registry url
+ */
+function get_base_url(protocol, host, prefix) {
```

Comment:
Changed to `combineBaseUrl`.
I was considering `combineBaseUrlFrom` or `baseUrlFrom` then example usage could looks like:
```javascript
let base = Utils.baseUrlFrom(req.protocol, req.get('host'), config.url_prefix);
```

I use such conventions from time to time (strict separation command from query) and found that it makes code more readable, but require more attention. So finally I keep it without `From` suffix.